### ISSUE TITLE
Feedback contains no information about the linux distribution, release and desktop environment

### ISSUE THREAD

**Issue by raffis** [gyselroth/balloon-client-desktop#114]
#### Description
**Is**:
Feedback os information only contains:
* "Linux" and kernel information

**Should**:
* Distribution name and version
* desktop environment and version

#### balloon desktop version
0.2.1

**Comment by raffis** [comment]
Those informations are required if the os is running a linux kernel:

distro name/version:
```
$ cat /etc/lsb-release
DISTRIB_ID=LinuxMint
DISTRIB_RELEASE=17.3
DISTRIB_CODENAME=rosa
DISTRIB_DESCRIPTION="Linux Mint 17.3 Rosa"
```

Kernel information:
```
$ cat /proc/version
Linux version 3.19.0-32-generic (buildd@lgw01-43) (gcc version 4.8.2 (Ubuntu 4.8.2-19ubuntu1) ) #37~14.04.1-Ubuntu SMP Thu Oct 22 09:41:40 UTC 2015
```

Desktop:
```
echo "$XDG_CURRENT_DESKTOP" (desktopEnvironment)
echo "$GDMSESSION" (gdmSession - Tells you what option you selected from the lightdm greeter to login.)
```

**Comment by raffis** [comment]
Probably a switch to https://www.npmjs.com/package/systeminformation makes sense. This will also solve  #135.

**Comment by raffis** [comment]
fixed by #146
